### ISSUE TITLE
Parser Error when using object spread (...)  operator in html templates

### ISSUE THREAD

**Issue by kle-pra** [angular-eslint/angular-eslint#2871]
**Description and reproduction of the issue**

Hi. Angular now supports object spread in template expressions, but @angular-eslint/template-parser fails to parse it and throws a parser error.

### Minimal reproduction
```ts
@Component({
  template: `
       @let a = { label: 'test', labelHidden: true };
       @let b = { ...a, type: 'number' };
  `
})
export class TestComponent {}
```
Error is:

> Parsing error: Parser Error: Unexpected token ., expected identifier, keyword, or string at column 3 in [{ ...a, type: 'number' }] 

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@angular-eslint/template-parser`  | `21.1.0` |
| `ESLint`                           | `9.39.2` |
| `node`                             | `24.11.0` |

```sh
# Please run `npx ng version` in your project and paste the full output here:
Angular CLI       : 21.1.1
Angular           : 21.1.1
Node.js           : 24.11.0
Package Manager   : npm 11.6.3
Operating System  : linux x64

┌───────────────────────────────────┬───────────────────┬───────────────────┐
│ Package                           │ Installed Version │ Requested Version │
├───────────────────────────────────┼───────────────────┼───────────────────┤
│ @angular-devkit/build-angular     │ 21.1.1            │ ^21.1.1           │
│ @angular/build                    │ 21.1.1            │ ^21.1.1           │
│ @angular/cli                      │ 21.1.1            │ 21.1.1            │
│ @angular/common                   │ 21.1.1            │ 21.1.1            │
│ @angular/compiler                 │ 21.1.1            │ 21.1.1            │
│ @angular/compiler-cli             │ 21.1.1            │ 21.1.1            │
│ @angular/core                     │ 21.1.1            │ 21.1.1            │
│ @angular/forms                    │ 21.1.1            │ 21.1.1            │
│ @angular/language-service         │ 21.1.1            │ 21.1.1            │
│ @angular/localize                 │ 21.1.1            │ ^21.1.1           │
│ @angular/platform-browser         │ 21.1.1            │ 21.1.1            │
│ @angular/platform-browser-dynamic │ 21.1.1            │ 21.1.1            │
│ @angular/router                   │ 21.1.1            │ 21.1.1            │
│ @angular/service-worker           │ 21.1.1            │ 21.1.1            │
│ rxjs                              │ 7.8.2             │ ^7.8.2            │
│ typescript                        │ 5.9.3             │ 5.9.3             │
│ zone.js                           │ 0.16.0            │ ~0.16.0           │
└───────────────────────────────────┴───────────────────┴───────────────────┘


```
- [x] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by sysmat** [comment]
Same here

**Comment by JamesHenry** [comment]
Should be fixed in https://github.com/angular-eslint/angular-eslint/releases/tag/v21.2.0
